Scripts/Python/nb01RPSGame.py

```diff
@@ -441,7 +456,14 @@ def _OnSitDown(self, state, seat, events):
         # Manage game state if standing up
         if state:
             if PtWasLocallyNotified(self.key):
+                """Disable Yeesha Book to prevent linking out
+                   Enable control keys to capture button presses or mouse movements
+                   Disable all movement keys so sit modifier wont trigger before we want it to
+                   Enable the mouse movement so we can move the camera and use the sides of the screen to trigger a movement"""
```

Comment:
These should be comments (lines beginning with `#`) instead of multiline strings, which are Python objects that the compyler has to deal with.